apps/pie-storybook/stories/testing/icons.test.stories.ts

```diff
@@ -0,0 +1,122 @@
+    <div style="--icon-size-override: 200px;">
+        <p>size: ${size}</p>
+        <p>--icon-size-override: 200px</p>
+        <icon-alcohol-filled size=${size}></icon-alcohol-filled>
+    </div>
+`;
+
+const AlcoholFilledLargeIconTemplate: TemplateFunction<LargeIconProps> = ({ size }) => html`
+    <p>For large icons, size must be a number greater than or equal to \`${largeIconSizeDefault}\`, and a multiple of \`${largeIconSizeModule}\`.</p>
+    <div class="c-iconGrid">
+        <icon-alcohol-filled-large size=${size}></icon-alcohol-filled-large>
+    </div>
+`;
+
+const AlcoholFilledLargeIconWithOverrideTemplate: TemplateFunction<LargeIconProps> = ({ size }) => html`
+    <div style="--icon-size-override: 200px;">
+        <p>size: ${size}px</p>
+        <p>--icon-size-override: 200px</p>
+        <icon-alcohol-filled-large size=${size}></icon-alcohol-filled-large>
+    </div>
+`;
+
+export const AlcoholFilledRegularIcon = createStory(AlcoholFilledRegularIconTemplate, defaultRegularIconProps)({}, {
+    layout: 'centered',
+    argTypes: {
+        size: {
+            control: 'select',
+            options: regularIconSizes,
+            defaultValue: {
+                summary: regularIconSizeDefault,
+            },
+        },
+    },
+});
+
+export const AlcoholFilledRegularIconWithOverride = createStory(AlcoholFilledRegularIconWithOverrideTemplate, defaultRegularIconProps)({}, {
+    layout: 'centered',
+    argTypes: {
+        size: {
+            control: 'select',
+            options: regularIconSizes,
+            defaultValue: {
+                summary: regularIconSizeDefault,
+            },
+        },
+    },
+});
+
+export const AlcoholFilledLargeIcon = createStory(AlcoholFilledLargeIconTemplate, defaultLargeIconProps)({}, {
+    layout: 'centered',
+    argTypes: {
+        size: {
+            control: { type: 'number', min: largeIconSizeDefault, step: largeIconSizeModule },
+            defaultValue: {
+                summary: largeIconSizeDefault,
+            },
+        },
+    },
+});
+
+export const AlcoholFilledLargeIconWithOverride = createStory(AlcoholFilledLargeIconWithOverrideTemplate, defaultLargeIconProps)({}, {
+    layout: 'centered',
+    argTypes: {
+        size: {
+            control: { type: 'number', min: largeIconSizeDefault, step: largeIconSizeModule },
+            defaultValue: {
+                summary: largeIconSizeDefault,
+            },
+        },
+    },
+});
+
+const regularIconPropMatrix : Partial<Record<keyof RegularIconProps, unknown[]>> = {
+    size: [...regularIconSizes, 'xxs', 128],
+};
+
+export const AlcoholFilledRegularIconVariations = createVariantStory<RegularIconProps>(AlcoholFilledRegularIconTemplate, regularIconPropMatrix);
+
+const largeIconPropMatrix : Partial<Record<keyof LargeIconProps, unknown[]>> = {
+    size: [16, largeIconSizeDefault, 64, 128, 250, 'xxl'],
+};
```

Comment:
Thought it might be useful to add some invalid sizes here to check the fallback works as expected, but happy to remove if others  think there's no value!